parser/transaction.go

```diff
@@ -461,9 +497,41 @@ func (tx *Transaction) ParseFromSlice(data []byte) ([]byte, error) {
 		}
 	}
 
-	if tx.version >= 4 && (spendCount+outputCount > 0) {
-		if !s.ReadBytes(&tx.bindingSig, 64) {
-			return nil, errors.New("could not read bindingSig")
+	if tx.version >= 4 {
+		if spendCount+outputCount > 0 && !s.ReadBytes(&tx.bindingSigSapling, 64) {
+			return nil, errors.New("could not read bindingSigSapling")
+		}
+	}
+	if tx.version >= 5 {
+		var actionsCount, proofsCount int
+		if !s.ReadCompactSize(&actionsCount) {
+			return nil, errors.New("could not read nActionsOrchard")
+		}
+		if !s.Skip(820 * actionsCount) {
+			return nil, errors.New("could not read vActionsOrchard")
+		}
+		if actionsCount > 0 {
+			if !s.Skip(1) {
+				return nil, errors.New("could not read flagsOrchard")
+			}
+			if !s.Skip(8) {
+				return nil, errors.New("could not read valueBalanceOrchard")
+			}
+			if !s.Skip(32) {
+				return nil, errors.New("could not read anchorOrchard")
+			}
+			if !s.ReadCompactSize(&proofsCount) {
+				return nil, errors.New("could not read sizeProofsOrchard")
+			}
+			if !s.Skip(proofsCount) {
+				return nil, errors.New("could not read proofsOrchard")
+			}
+			if !s.Skip(64 * actionsCount) {
+				return nil, errors.New("could not read vSpendAuthSigsOrchard")
```

Comment:
```suggestion
				return nil, errors.New("could not skip vSpendAuthSigsOrchard")
```

and similar for the other `Skip` calls.